src/Checkout.js

```diff
@@ -1,10 +1,40 @@
-import { PayPalButtons } from "@paypal/react-paypal-js";
+import { PayPalButtons, usePayPalScriptReducer } from "@paypal/react-paypal-js";
+import { useState } from "react";
+import loading from './three-dots.svg';
+
+function Checkout(props) {
+
+  const [{ options, isPending,  isInitial}, dispatch] = usePayPalScriptReducer();
+  const [currency, setCurrency] = useState(options.currency);
+
+  const saveDeferLoadingHandler = () => {
```

Comment:
Hi @AnuragVathalurin can you push up the new changes we discussed? After that we can merge it up.